Update MITM to Machine-in-the-Middle (#3175)
OWASP/owasp-mastg: tests/android/MASVS-NETWORK/MASTG-TEST-0022.md

   function errorCallback(message) {
     alert(message);
     if (message === "CONNECTION_NOT_SECURE") {

>        // There is likely a machine in the middle attack going on, be careful!

Reviewer: Capitalization and dashes missing

>        // There is likely a machine in the middle attack going on, be careful!

Author: ```suggestion
       // There is likely a Machine-in-the-Middle attack going on, be careful!
```

>        // There is likely a Machine-in-the-Middle attack going on, be careful!

Reviewer: ```suggestion
       // There is likely a MITM attack going on, be careful!
```
